Translation Editor/translation_ru.json

```diff
@@ -127,71 +129,71 @@
 				"Автоповорот",
 				"экрана"
 			],
-			"desc": "Ориентация дисплея <A. Автоматический, Л. Левая рука, П. Правая рука>"
+			"desc": "Ориентация дисплея <A = Авто, Л = Левая рука, П = Правая рука>"
 		},
 		"BoostEnabled": {
 			"text": "Турбо",
 			"text2": [
 				"Режим",
 				"Турбо"
 			],
-			"desc": "Турбо-режим при удержании кнопки А при пайке "
+			"desc": "Активирует "Турбо-режим" при удержании кнопки А во время пайки"
```

Comment:
You can't use the quotes inline like this, they need to be escaped

Reply:
https://github.com/Ralim/ts100/pull/563/commits/abf3394931e21974ec9ed5739673bf5c1e12df38